ssl/statem/extensions.c

```diff
@@ -1449,7 +1449,11 @@ static int final_key_share(SSL_CONNECTION *s, unsigned int context, int sent)
                     group_id = pgroups[i];
 
                     if (check_in_list(s, group_id, clntgroups, clnt_num_groups,
-                                      1))
+                                      1)
```

Comment:
Does line-length allow for this to be put with the previous line?

Reply:
No - line-length prevents this